src/mono/mono/component/debugger-agent.c

```diff
@@ -6796,7 +6796,6 @@ static void
 get_types (gpointer key, gpointer value, gpointer user_data)
 {
 	MonoAssembly *ass;
-	gboolean type_resolve;
```

Comment:
This looks like a bug; `type_resolve` is uninitialized and this value is both read and written. What is the intended behavior here?